src/styles/index.scss

```diff
@@ -1,3 +1,71 @@
-body {
+@import 'variables';
+
+* {
   margin: 0;
+  padding: 0;
+}
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), using the universal selector (*) for zeroing margins and paddings is not recommended due to its heavy performance cost. It's better to use specific elements or a class.